### PR TITLE
Jira, Github, GHE Drafts: Early adopters edition

### PR DESCRIPTION
**Pull request by saragilford**
Docs for some of our new integrations. Pictures to come in the next few days, but want to have instructions live for when we contact early adopters tomorrow morning.

### CODE REVIEW

File: src/collections/_documentation/integrations/github.md [hunk 1]
```diff
@@ -2,19 +2,56 @@
 title: GitHub
 sidebar_order: 2
 ---
+Sentry’s new Github integration has the following features: commit tracking, issue management, and Github SSO.You can now use the data from your commits to Github to help you find and fix bugs faster.
```
Comment (markstory) [inline src/collections/_documentation/integrations/github.md]: Should you have a space after the `Github SSO.`?  GitHub also prefers to have the `H` capitalized. All their [press materials](https://github.com/logos) use big H.

File: src/collections/_documentation/integrations/github.md [hunk 2]
```diff
@@ -2,19 +2,56 @@
...
 
-Sentry provides two ways to integrate with GitHub.
+## Configure Github
 
-## GitHub SSO
 
-[Enable Single Sign-on]({%- link _documentation/learn/sso.md -%})
+1. In Sentry, navigate to Organization Settings > Integrations. *Note: only users with Owner and Manager permissions will have access to this page.*
+2. If you have the legacy Github integration installed, you’ll see a button next to Github that says ‘Upgrade.’ If you do not have the legacy Github integration installed, you’ll see a button that says ‘Install.’ Click this button.
+3. In the resulting modal, click ‘Add Installation.’
```
Comment (markstory) [inline src/collections/_documentation/integrations/github.md]: Should the `.` be inside the quotes?

File: src/collections/_documentation/integrations/githubenterprise.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+---
+title: GitHub Enterprise
+sidebar_order: 3
+---
+Sentry’s new Github Enterprise integration has the following features: commit tracking and issue management..You can now use the data from your commits to Github to help you find and fix bugs faster.
```
Comment (markstory) [inline src/collections/_documentation/integrations/githubenterprise.md]: You have a double `.` after issue management.

**Comment by cameronmcefee** [comment]
I haven't read the actual content but LGTM as far as content goes. Can we rename the GitHub Enterprise doc to be github-enterprise so there'll be a dash in the url?

**Pull request merged**